packages/editor/src/lib/editor/Editor.ts

```diff
@@ -5133,7 +5133,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 
 		for (const id of ids) {
 			const shape = this.getShape(id)!
-			const localDelta = Vec.Cast(offset)
+			const localDelta = Vec.From(offset)
```

Comment:
i wonder if `Cast` should be consistent on whether it returns a new object or the same object in memory. maybe `Cast` should always return a new `A` to avoid this pitfall? not sure